### PR TITLE
fix: add random_state to train_test_split to ensure reproducibility

### LINKED ISSUE #515: Bug: train_test_split() called without random_state — results are non-reproducible
## Bug Report

**Description**
Multiple projects call `train_test_split()` without setting `random_state`, meaning the
train/test split changes on every run. This produces different model accuracy each run,
making results impossible to reproduce or compare — especially bad in a learning repository
where students expect consistent output.

## Affected Files

| File | Line |
|------|------|
| `Bitcoin Price Prediction Web App/app.py` | 50 |
| `Chicken_Disease_Classification/research/mlflow_exp/example.py` | 47 |
| `Social Media Fake Accounts Detection/jserver/model.py` | 76 |
| `Ensemble Methds in ML/voting.py`, `blending.py`, `stacking.py` | various |

## Code (Buggy)
```python
xtrain, xtest, ytrain, ytest = train_test_split(X, Y, test_size=0.2)
```

## Fix
```python
xtrain, xtest, ytrain, ytest = train_test_split(X, Y, test_size=0.2, random_state=42)
```

## Severity
Medium — non-reproducible experiments; misleads beginners learning ML

### PR DESCRIPTION
**Pull request by Daksh7785**
Add random_state=42 to train_test_split calls in multiple ML projects

Closes #515

## Summary

Closes #

## Type of Change

- [ ] Bug fix
- [ ] New feature
- [ ] Documentation update
- [ ] Refactor
- [ ] Tests
- [ ] Infra / CI

## Checklist

- [ ] I have read [CONTRIBUTING.md](../CONTRIBUTING.md)
- [ ] My code follows the project style (PEP 8 for Python, ESLint for TS)
- [ ] I have added/updated tests where relevant
- [ ] `pytest backend/tests/` passes locally
- [ ] I have not committed `.env` or any secrets
- [ ] I have updated documentation if needed

### CODE REVIEW

**Comment by SdSarthak** [comment]
This PR adds a Bitcoin price prediction Streamlit app (`Bitcoin Price Prediction Web App/app.py`) to the AegisAI repository. This is an entirely different project and has no relation to EU AI Act compliance tooling. Closing as wrong repository.
